src/backend/distributed/shared_library_init.c

```diff
@@ -1374,6 +1374,20 @@ RegisterCitusConfigVariables(void)
 		GUC_NO_SHOW_ALL,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.enable_schema_based_sharding",
```

Comment:
I'm kind of wondering whether this should be schema_based_sharding or schema_sharding.

I do feel like schema_based_sharding sounds a bit better, but if we ever add database sharding, then database_based_sharding sounds much worse than database_sharding. :)